spec.html

```diff
@@ -6367,12 +6370,15 @@ <h1>
       <emu-alg>
         1. Assert: _O_ is an ordinary, extensible object with no non-configurable properties.
         1. Let _newDesc_ be the PropertyDescriptor { [[Value]]: _V_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
-        1. Perform ! DefinePropertyOrThrow(_O_, _P_, _newDesc_).
+        1. Perform ! _O_.[[DefineOwnProperty]](_P_, _newDesc_).
```

Comment:
I find this reduces clarity; calling an AO a better than calling an internal method.

Reply:
Thanks, I'll keep that in mind :+1: